test/acceptance/features/volumeMount.feature

```diff
@@ -0,0 +1,166 @@
+                    name: backend-demo
+
+                application:
+                    name: generic-app-a-d-u
+                    group: apps
+                    version: v1
+                    resource: deployments
+            """
+        Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-backend" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-backend" should be changed to "True"
+        And The env var "BACKEND_HOST" is not available to the application
+        And The env var "BACKEND_PORT" is not available to the application
+        And Content of file "/bindings/binding-backend/BACKEND_HOST" in application pod is
+            """
+            example.common
+            """
+        And Content of file "/bindings/binding-backend/BACKEND_PORT" in application pod is
+            """
+            8080
+            """
+
+    Scenario: Binding is injected as file into application pod at overriden location
+        Given OLM Operator "backend" is running
+        * Generic test application "generic-app-a-d-u" is running
+        * The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: backend-demo
+                annotations:
+                    "service.binding/host": "path={.spec.host}"
+                    "service.binding/port": "path={.spec.port}"
+            spec:
+                host: example.common
+                port: 8080
+            """
+        When Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-backend
+            spec:
+                mountPath: "/foo/bar"
+                bindAsFiles: true
+                services:
+                -   group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: backend-demo
+
+                application:
+                    name: generic-app-a-d-u
+                    group: apps
+                    version: v1
+                    resource: deployments
+            """
+        Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-backend" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-backend" should be changed to "True"
+        And The env var "BACKEND_HOST" is not available to the application
+        And The env var "BACKEND_PORT" is not available to the application
+        And Content of file "/foo/bar/BACKEND_HOST" in application pod is
+            """
+            example.common
+            """
+        And Content of file "/foo/bar/BACKEND_PORT" in application pod is
```

Comment:
I am not sure if bindings should be injected under the name different from the binding name - spec does not say that we are allowed to change the filename. I would expect that the filename is `/foo/bar/host`, what is the purpose of `BACKEND_` prefix?

Reply:
In the spec, there will be only one service at a time. But in SBO there are multiple services. So, the chance for a name collision is high. Especially common names like `host`, `port` will be there for most of the services.

Reply:
Do the secret key names have the BACKEND  _prefix regardless?

If yes, I think you could get rid of the ootb prefix by setting spec.services.envVarPrefix = ""

Reply:
I would suggest implement this in another PR.

Reply:
I don't think anything needs to be done from a code perspective. It should already work, we should have a test that verifies the behaviour.